aws-assume-role-policy/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "project" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>variable "project" is declared but not used